builds/misc/images-release-base-image-update.yaml

```diff
@@ -0,0 +1,575 @@
+name: $(version)
+
+stages:
+################################################################################
+  - stage: BuildExecutables
+################################################################################
+    displayName: Build Executables
+    pool:
+      name: $(pool.linux.name)
+      demands:
+        - ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker
```

Comment:
This should be 20.04 to have parity with https://github.com/Azure/iotedge/blob/release/1.2/builds/misc/images-release.yaml